### PR TITLE
Please add new TuYa TV02-Zigbee Thermostat support.

### PR DESCRIPTION
**Pull request by vladi1234**

### CODE REVIEW

File: adapters/tuya/__init__.py [hunk 1]
```diff
@@ -3,6 +3,7 @@
 from adapters.tuya.TS0013 import TS0013
 from adapters.tuya.TS0015 import TS0015
 from adapters.tuya.TS0601 import TS0601
+from adapters.tuya.TV02 import TV02
```
Comment (stas-demydiuk) [inline adapters/tuya/__init__.py:6]: I don't see this file in the PR, could you please double-check that all required changes are included

**Comment by vladi1234** [comment]
@stas-demydiuk Thank you. I uploaded again.

**Comment by stas-demydiuk** [comment]
Thanks!

**Comment by vladi1234** [comment]
Thanks!

**Pull request merged**